draft-ietf-rats-eat.md

```diff
@@ -818,6 +818,50 @@ uptime-claim = (
 )
 ~~~~
 
+## The Context Claim (context)
+
+EAT's may be used in the context of several different applications.  The context
+claim provides an indication to an EAT consumer about  the intended usage
+of the token.
```

Comment:
Maybe add "This claim can be used as a way for an application using EAT to internally distinguish between different ways it uses EAT".

Reply:
Added statement.  Please check.